ifrs17/Constants/Validations.ipynb

```diff
@@ -61,45 +56,15 @@
         {
             "cell_type": "code",
             "source": [
-                "public enum Error { ",
-                "\n    // Import",
-                "\n    NoMainTab, IncompleteMainTab, ParsingInvalidOrScientificValue, ValueTypeNotFound, ValueTypeNotValid, ",
-                "\n    ReportingNodeInMainNotFound, YearInMainNotFound, MonthInMainNotFound, ScenarioInMainNotAvailable,",
-                "\n    AocTypeNotValid, AocTypeCompulsoryNotFound, AocTypePositionNotSupported, AocConfigurationOrderNotUnique,",
-                "\n    AccidentYearTypeNotValid,",
-                "\n    // Partition",
-                "\n    PartitionNotFound, ParsedPartitionNotFound, PartititionNameNotFound, PartitionTypeNotFound,",
-                "\n    // Dimensions",
-                "\n    AmountTypeNotFound, EstimateTypeNotFound, ReportingNodeNotFound, AocTypeMapNotFound, AocTypeNotFound, PortfolioGicNotFound, PortfolioGricNotFound, ",
-                "\n    InvalidAmountTypeEstimateType, MultipleTechnicalMarginOpening, DimensionNotFound, NoScenarioOpening,",
-                "\n    // Exchange Rate",
-                "\n    ExchangeRateNotFound, ExchangeRateCurrency,",
-                "\n    // Data Note State",
-                "\n    ChangeDataNodeState, InactiveDataNodeState,",
-                "\n    // Parameters",
-                "\n    ReinsuranceCoverageDataNode, DuplicateInterDataNode, DuplicateSingleDataNode, MissingSingleDataNodeParameter, InvalidDataNode, InvalidDataNodeForOpening, InvalidCashFlowPeriodicity, MissingInterpolationMethod, InvalidInterpolationMethod, InvalidEconomicBasisDriver, InvalidReleasePattern,",
-                "\n    // Storage",
-                "\n    DataNodeNotFound, PartnerNotFound, PeriodNotFound, RatingNotFound, CreditDefaultRateNotFound, MissingPremiumAllocation, ReinsuranceCoverage, ",
-                "\n    YieldCurveNotFound, YieldCurvePeriodNotApplicable, EconomicBasisNotFound, AccountingVariableTypeNotFound, InvalidGric, InvalidGic, ReleasePatternNotFound,",
-                "\n    MissingPreviousPeriodData,",
-                "\n    // Scopes",
-                "\n    NotSupportedAocStepReference, MultipleEoP,",
-                "\n    // Data completeness",
-                "\n    MissingDataAtPosting, MissingCombinedLiability, MissingCoverageUnit, ",
-                "\n    // Index",
-                "\n    NegativeIndex,",
-                "\n    // Default",
-                "\n    Generic",
-                "\n};"
-            ],
-            "metadata": {},
-            "execution_count": 0,
-            "outputs": []
-        },
-        {
-            "cell_type": "markdown",
-            "source": [
-                "# Messages"
+                "public abstract class ValidationBase{",
+                "\n    public string InternalValue {get; protected set;}",
```

Comment:
Instead of InternalValue I prefer ErrorCode : rename every occurrence, thanks!

Reply:
It is a little bit confusing name, because this variable applies both to the error and the warning. How about MessageCode or InternalCode or simply Code?

Reply:
I like MessageCode. 
Thanks, I didn't realize the confusion...

Reply:
Changed to MessageCode everywhere